airflow/models/baseoperator.py

```diff
@@ -309,7 +309,7 @@ def __init__(
         priority_weight=1,  # type: int
         weight_rule=WeightRule.DOWNSTREAM,  # type: str
         queue=conf.get('celery', 'default_queue'),  # type: str
-        pool=Pool.DEFAULT_POOL_NAME,  # type: str
+        pool=None,  # type: str
```

Comment:
@vshshjn7 will setting pool to None here cause issues?

Reply:
This is done as a part of https://github.com/apache/airflow/pull/8587